src/error.rs

```diff
@@ -35,10 +35,14 @@ impl Error {
     pub(crate) const RTL_GEN_RANDOM_FAILED: Error = internal_error!(4);
     pub(crate) const FAILED_RDRAND: Error = internal_error!(5);
     pub(crate) const NO_RDRAND: Error = internal_error!(6);
-    pub(crate) const BINDGEN_CRYPTO_UNDEF: Error = internal_error!(7);
-    pub(crate) const BINDGEN_GRV_UNDEF: Error = internal_error!(8);
-    pub(crate) const STDWEB_NO_RNG: Error = internal_error!(9);
-    pub(crate) const STDWEB_RNG_FAILED: Error = internal_error!(10);
+    #[doc(hidden)]
+    pub const BINDGEN_CRYPTO_UNDEF: Error = internal_error!(7);
```

Comment:
Why hide these? They are defined constants either way.

Reply:
I'll need to go back an document these (this was just to make the CI happy). There's also the separate question of if we should expose all of the `Error` constants or just those necessary to make the custom RNGs work. What do you think?

Reply:
There's also the issue of `Debug`/ `Display` implementations (error strings) when moving implementations out of the main crate.

Reply:
I'm not sure what you mean? The error strings would stay in the main `getrandom`, I guess it means Custom RNGs not in this repo just have to use numbers instead of words. That's not ideal, but fixing it is probably just too hard.

Reply:
Fixing it is https://github.com/rust-random/rand/issues/837 which it looks like we will not solve. But for these WASM crates we should keep the error stings in `getrandom`.

Unless we add an auxilliary extern Rust function:
```rust
fn __getrandom_error_msg(code: NonZeroU32) -> Option<&'static str>;
```
I don't believe it's worth it for now though (and it doesn't solve error messages for anything outside `getrandom`).